### PR TITLE
update doc of shape op

### PR DESCRIPTION
**Pull request by JZ-LIANG**
### PR types
Others

### PR changes
Docs

### Describe
update doc of shape op

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -11257,28 +11257,19 @@ def shape(input):
...
-            inputs = fluid.data(name="x", shape=[3, 100, 100], dtype="float32")
-            output = fluid.layers.shape(inputs)
-
-            exe = fluid.Executor(fluid.CPUPlace())
-            exe.run(fluid.default_startup_program())
-
-            img = np.ones((3, 100, 100)).astype(np.float32)
+            import paddle
+            x = paddle.randn((2,3))
+            print(x.shape) # [2, 3]
```
Comment (zhiqiu) [inline python/paddle/fluid/layers/nn.py]: Call the API in example code, like `print(paddle.shape(x))`

Comment (zhiqiu) [reply python/paddle/fluid/layers/nn.py]: Write a dygraph branch in function implementation and call `core.ops`.

Comment (JZ-LIANG) [reply python/paddle/fluid/layers/nn.py]: done

**Review by zhiqiu — APPROVED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request closed without merging**